.travis.yml

```diff
@@ -52,6 +53,11 @@ jobs:
     - PACKAGE=coq-reglang.dev
     - NJOBS=2
     <<: *OPAM
+  - env:
+    - COQ_IMAGE=coqorg/coq:8.12
```

Comment:
Is this an image that follows the v8.12 release branch of the coq repositiory and will eventually become the image for the v8.12 release?

Reply:
For precision, it currently follows `v8.12`, will then switch to `8.12+beta1`, and then finally `8.12.0`. So short answer yes.